src/Microsoft.Identity.Web/LoggingEventId.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Identity.Web
+{
+    /// <summary>
+    /// EventIds for Logging.
+    /// </summary>
+    internal static class LoggingEventId
+    {
+        // DistributedCacheAdapter EventIds 100+
```

Comment:
> // DistributedCacheAdapter EventIds 100+ [](start = 7, length = 41)

nit: the other side of the #pragma

Reply:
i'm not sure what you are saying, but my cat has an idea.